modules/integration_azure-storage-account/conf/07-latency-e2e.yaml

```diff
@@ -0,0 +1,17 @@
+module: "Azure Storage Account"
```

Comment:
Having a global latency status detector does not make sense in my opinion. We may have big variations depending if we use tables or files.